### PR TITLE
Update to Minecraft 1.8

### PR DESCRIPTION
**Pull request by Techcable**
This uses the player list packet which contains all player info on versions after 1.8

Not Compatible with versions < 1.8
Not Compatible with Protocol Hack

### CODE REVIEW

File: src/main/java/net/md_5/itag/iTag.java [hunk 1]
```diff
@@ -46,15 +51,27 @@ public void onEnable()
...
         {
             @Override
             public void onPacketSending(PacketEvent event)
             {
-                event.getPacket().getGameProfiles().write( 0, getSentName( event.getPacket().getIntegers().read( 0 ), event.getPacket().getGameProfiles().read( 0 ), event.getPlayer() ) );
+                if (event.getPacket().getPlayerInfoAction().read(0) != PlayerInfoAction.ADD_PLAYER) return;
+		List<PlayerInfoData> newPlayerInfoDataList = new ArrayList<>();	
+		List<PlayerInfoData> playerInfoDataList = event.getPacket().getPlayerInfoDataLists().read(0);
+		for (PlayerInfoData playerInfoData : playerInfoDataList) {
+		    if (playerInfoData == null || playerInfoData.getProfile() == null || Bukkit.getPlayer(playerInfoData.getProfile().getUUID()) == null) newPlayerInfoDataList.add(playerInfoData); //Unknown Player
```
Comment (md-5) [inline src/main/java/net/md_5/itag/iTag.java]: You call getPlayer twice, here and in the getEntityId method, just store the bukkit player in the loop

Comment (Techcable) [inline src/main/java/net/md_5/itag/iTag.java]: Done

File: src/main/java/net/md_5/itag/iTag.java [hunk 2]
```diff
@@ -46,12 +51,24 @@ public void onEnable()
...
             @Override
             public void onPacketSending(PacketEvent event)
             {
-                event.getPacket().getGameProfiles().write( 0, getSentName( event.getPacket().getIntegers().read( 0 ), event.getPacket().getGameProfiles().read( 0 ), event.getPlayer() ) );
+                if (event.getPacket().getPlayerInfoAction().read(0) != PlayerInfoAction.ADD_PLAYER) return;
+                List<PlayerInfoData> newPlayerInfoDataList = new ArrayList<>();	
+                List<PlayerInfoData> playerInfoDataList = event.getPacket().getPlayerInfoDataLists().read(0);
+                for (PlayerInfoData playerInfoData : playerInfoDataList) {
+                    Player player = Bukkit.getPlayer(playerInfoData.getProfile().getUUID());
+                    if (playerInfoData == null || playerInfoData.getProfile() == null || player == null) { //Unknown Player
```
Comment (md-5) [inline src/main/java/net/md_5/itag/iTag.java]: Player must go below the null checks.

**Comment by Techcable** [comment]
@md-5 
Are you going to review more?

**Comment by Techcable** [comment]
@md-5 
Bump

**Comment by md-5** [comment]
Changed and added

**Pull request closed without merging**